docs/_data/curio_scan.yaml

```diff
@@ -7,6 +7,9 @@ options:
     - name: context
       usage: |
         Expand context of schema classification e.g., --context=health, to include data types particular to health
+    - name: data-subject-mapping
+      usage: |
+        Override default data subject mapping by providing a path to a custom mapping JSON file
```

Comment:
@markmichon some context here: adding a new flag so users can override the default data subject mapping with their own JSON file in this format https://github.com/Bearer/curio/blob/main/pkg/classification/db/subject_mapping.json

Reply:
The flag text looks fine. I think we're going to need a better format example than that, specifically what is allowed other than `user`. Is this documented internally anywhere or do we validate it?  

And we'll need some extra details on why someone might want to do it once the privacy report is up and running.

Reply:
Thanks @markmichon 🙏 

> Is this documented internally anywhere or do we validate it?

Right now there's no validation, so you'd be free to put any value/s in place of "User". We also don't validate that you've given us a sensible JSON file, which is maybe something we'd want to add too 🤔